projects/cds/src/profiletag/model/profile-tag-lifecycle.ts

```diff
@@ -0,0 +1,7 @@
+import { Observable } from 'rxjs';
+import { ConsentChangedPushEvent } from './profile-tag.model';
+
+export abstract class ProfileTagLifecycle {
```

Comment:
What is the idea behind this abstract class?

Reply:
removed the abstract class